src/main/java/com/flair/bi/compiler/mysql/MySQLListener.java

```diff
@@ -199,6 +199,10 @@ else if(Optional.ofNullable(ctx.func_call_expr()).isPresent() && "YEARQUARTER".e
                 .map(property::get)
                 .ifPresent(str::append);
 
+        if (str.length() == 0) {

```

Comment:
Use String#isEmpty() instead.

Reply:
StringBuilder does not have `isEmpty` method afaik.